weave-js/src/components/Panel2/panelTree.ts

```diff
@@ -12,10 +12,17 @@ The UI state is a tree of panels. There are three types of non-leaf panels:
 */
 
 import {
```

Comment:
I'd like to spend some time at next tech office hours or 1-1 to understand panelTree a bit more